src/bankAccount.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount) {
+  this.name = name;
+  this.amount = amount;
+  this.history = ['Initial: ' + amount];
+}
+
+BankAccount.prototype = {
+  getInfo() {
+    return ('Name: ' + this.name + ', Amount: ' + this.amount + '$');
+  },
```

Comment:
не переписуй BankAccount.prototype, тому що так ти втрачаєш властивість constructor
```suggestion
BankAccount.prototype.getInfo = function() {
    return ('Name: ' + this.name + ', Amount: ' + this.amount + '$');
  },
```